### PR TITLE
Bundle Bower Dependencies

### PR DESCRIPTION
**Pull request by thesupremecommander**

### CODE REVIEW

File: lib/bundles/parser.js [hunk 1]
```diff
@@ -111,6 +113,24 @@ function installNpmPackages(bundle) {
     return deferred.promise;
 }
 
+function installBowerPackages(bundle) {
+    var deferred = Q.defer();
+
+    // Do nothing if bower.json does not exist
+    var packagejsonPath = path.join(bundle.dir, 'bower.json');
+    if (!fs.existsSync(packagejsonPath)) {
+        log.trace('Nothing to install for bundle', bundle.name);
```
Comment (Lange) [inline lib/bundles/parser.js]: This trace message should specify that we're talking about bower.

**Comment by Lange** [comment]
This looks pretty good. Can you add tests for it?

**Comment by MattMcNam** [comment]
Bower components can be used by views as well, wondering if there's a better url than `/dashboard/:bundle/components`?

**Comment by Lange** [comment]
@MattMcNam They're accessible from both `/dashboard/:bundle/components` and `/view/:bundle/components`. This way, relative paths work nicely in both places.

**Comment by thesupremecommander** [comment]
I added a test for making sure that the installation works properly, similar to the npm test. Is there anything else that I should add a test for?

**Comment by Lange** [comment]
@thesupremecommander A test to make sure that the files are accessible from both endpoints would be neat.

**Comment by MattMcNam** [comment]
Oh, I missed that! Don't mind me then I'm happy with those paths :+1: 

And yeah, a simple test that ensures both paths work should be all that we need for now.

**Pull request merged**